packages/react-components/src/hooks/useAlarms/utils/alarmDataUtils.ts

```diff
@@ -27,10 +27,19 @@ export const buildFromAssetResponse = ({
   status: AlarmDataStatus;
   assetResponse?: DescribeAssetResponse;
 }): AlarmData[] => {
-  const { assetId, assetCompositeModelId, inputPropertyId } = request;
+  const { assetId, assetCompositeModelId, inputPropertyId, assetModelId } =
+    request;
 
   if (!assetResponse || !assetId) {
-    return [];
+    const alarmData = buildFromCompositeModel(undefined, status);
```

Comment:
Do we need to call `buildFromCompositeModel` here? We essentially only need to add the status field.